desc/protoparse/ast/enum.go

```diff
@@ -0,0 +1,99 @@
+package ast
+
+import "fmt"
+
+type EnumNode struct {
+	compositeNode
+	Keyword    *KeywordNode
+	Name       *IdentNode
+	OpenBrace  *RuneNode
+	Options    []*OptionNode
+	Values     []*EnumValueNode
+	Reserved   []*ReservedNode
+	CloseBrace *RuneNode
+
+	AllDecls []EnumElement
```

Comment:
Is this duplicative of what is in EnumNode?

Reply:
This declaration _is_ in EnumNode. I don't understand the question.

Reply:
I think I understand the question now -- it _was_ duplicative, a different representation of the decls above, between `OpenBrace` and `CloseBrace`. But I've changed it now so that we _only_ have `Decls` and the other slices, that categorized declarations by type, are no more.